app.js

```diff
@@ -13,44 +13,56 @@ try {
 
 if (!!majorVersion && majorVersion < 14) {
   const octoFarmServer = setupFallbackExpressServer();
-  return serveNodeVersionFallback(octoFarmServer);
-}
+  serveNodeVersionFallback(octoFarmServer);
+} else {
+  const {
+    setupExpressServer,
+    serveOctoFarmNormally,
+    ensureSystemSettingsInitiated
+  } = require("./app-core");
+  const {
+    setupEnvConfig,
+    fetchMongoDBConnectionString,
+    fetchOctoFarmPort
+  } = require("./app-env");
 
-const {
-  setupExpressServer,
-  serveOctoFarmNormally,
-  ensureSystemSettingsInitiated
-} = require("./app-core");
-const {
-  setupEnvConfig,
-  fetchMongoDBConnectionString,
-  fetchOctoFarmPort
-} = require("./app-env");
+  const Logger = require("./server_src/lib/logger.js");
+  const mongoose = require("mongoose");
+  const logger = new Logger("OctoFarm-Server");
 
-const Logger = require("./server_src/lib/logger.js");
-const mongoose = require("mongoose");
-const logger = new Logger("OctoFarm-Server");
+  function bootAutoDiscovery() {
+    require("./server_src/runners/autoDiscovery.js");
+  }
 
-function bootAutoDiscovery() {
-  require("./server_src/runners/autoDiscovery.js");
-}
+  setupEnvConfig();
+  const octoFarmServer = setupExpressServer();
 
-setupEnvConfig();
-const octoFarmServer = setupExpressServer();
-
-mongoose
-  .connect(fetchMongoDBConnectionString(), {
-    useNewUrlParser: true,
-    useUnifiedTopology: true,
-    useFindAndModify: false,
-    useCreateIndex: true,
-    serverSelectionTimeoutMS: 2500
-  })
-  .then(() => ensureSystemSettingsInitiated())
-  .then(() => serveOctoFarmNormally(octoFarmServer, fetchOctoFarmPort()))
-  .catch(async (err) => {
-    logger.error(err);
-    serveDatabaseIssueFallback(octoFarmServer, fetchOctoFarmPort());
-  });
-
-bootAutoDiscovery();
+  mongoose
+    .connect(fetchMongoDBConnectionString(), {
+      useNewUrlParser: true,
+      useUnifiedTopology: true,
+      useFindAndModify: false,
+      useCreateIndex: true,
+      serverSelectionTimeoutMS: 2500
+    })
+    .then(() => ensureSystemSettingsInitiated())
+    .then(async () => {
+      const port = fetchOctoFarmPort();
+      if (!port || Number.isNaN(parseInt(port))) {
+        throw new Error(
+          "The server database-issue mode requires a numeric port input argument"
+        );
+      }
+
+      const app = await serveOctoFarmNormally(octoFarmServer);
+      app.listen(port, "0.0.0.0", () => {
```

Comment:
Listening is done here for the normal serving. The fallbacks are still listening in the app-core file (node version, database connection issues)